### ISSUE TITLE
Doesn't work with Soundcloud on Chrome

### ISSUE THREAD

**Issue by mrfoto** [mmth/simplify_web_suite#6]
Apparently I'm not the only one https://chrome.google.com/webstore/detail/simplify/caopaadcaikafcghcgohcfjccglpbmdl/support

**Comment by din** [comment]
Hello,

This is a known issue with secure web-sockets while using Google Chrome. You can temporarily solve the issue by doing the following:

1) Open chrome://flags in the address bar.
2) Enable the Chrome flag Allow insecure WebSocket from https origin.
3) Restart the Chrome browser.
4) Try using SoundCloud with Simplify again.

We are aware of the issue and we will include secure web-sockets support in the upcoming version of Simplify.

**Comment by mrfoto** [comment]
Yup, it works! Maybe you should post this to the support tab of extension

**Comment by Kl0tl** [comment]
Hi, thanks for your work on Simplify ! Is there anything we can do to help on this ?
